src/Task/Psalm/services.yaml

```diff
@@ -0,0 +1,8 @@
+services:
```

Comment:
This is not related to current PR. It's something I missed in the GrumpPHP 2.5 update - just a nasty error:
![image](https://github.com/wunderio/code-quality/assets/492375/4df79b7e-50bd-450b-9b26-030b2fa02fbb)